be/src/util/cityhash102/city.cc

```diff
@@ -33,8 +33,7 @@
 #include <algorithm>
 #include <string.h>  // for memcpy and memset
 
-using namespace std;
-
+//using namespace std;
```

Comment:
just del the useless code, do not comment

Reply:
thx for the reminder. fixed them.